### PR TITLE
[PT-BR] Beatmap Spotlights

### PR DESCRIPTION
**Pull request by Flantsune**
Fully translated Beatmap_Spotlights in pt-br

Unfortunately kind of late for Summer 2020 season but should be useful as a base for future translation updates

### CODE REVIEW

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+  - Charts
+  - Ranking Charts
+  - Seasonal Spotlights
+---
+
+# Beatmap Spotlights
+
+![Beatmap Spotlights logo](img/logo.jpg)
+
+Os **Beatmap Spotlights** (também conhecidos como *Spotlights* ou *Ranking Charts*) é um projeto de curadoria recorrente para recomendação e destaque de [beatmaps](/wiki/Beatmaps) pela seu design e gameplay excelentes e únicos. É acompanhado por uma [liga de temporada](#spotlights-seasons) que dá recompensas para todos os jogadores participantes.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:12]:
```suggestion
O **Beatmap Spotlights** (também conhecido como *Spotlights* ou *Ranking Charts*) é um projeto de curadoria recorrente para recomendação e destaque de [beatmaps](/wiki/Beatmaps) por seu design e gameplay excelentes e únicos. É acompanhado por uma [liga de temporada](#spotlights-seasons) que dá recompensas para todos os jogadores participantes.
```
``Beatmap spotlights`` it's just the name of one project so I think it should be singular and `pela seu design` seems strange to me, just a suggestion tho

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+
+**A iteração atual desse projeto está em desenvolvimento e muitos de suas funcionalidades ainda estão sendo implementadas.** Específicos desse sistema, tais como recompensas, rankings sazonais ou funcionalidades de jogabilidade, podem mudar a qualquer momento.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:14]:
```suggestion
**A iteração atual desse projeto está em desenvolvimento e muitas de suas funcionalidades ainda estão sendo implementadas.** Específicos desse sistema, tais como recompensas, rankings sazonais ou funcionalidades de jogabilidade, podem mudar a qualquer momento.
```
`muitos` is referring to `funcionalidades` and it is a female noun

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+
+A temporada atual é a temporada de Verão 2020. É considerada a ser o primeiro teste do novo sistema de Beatmap Spotlights.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:16]:
```suggestion
A temporada atual é a temporada de Verão 2020. É considerada o primeiro teste do novo sistema de Beatmap Spotlights.
```
Ease of comprehension

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+
+A Colocação de Temporada conta todo os resultados semanais de Pontuação Ranqueada de cada participante. Baseado na Colocação de Temporada, cada participante é atribuído com uma divisão de liga refletindo a sua posição relativa dentro da colocação.
+
+1. A pontuação da colocação de temporada é uma soma ponderada de toda a Pontuação Ranqueada Semanal adquirida em salas permanentes.
+2. Participantes poderão possuir apenas uma Pontuação Ranqueada Semanal por playlist.
+   - Jogar a mesma playlist em outra semana irá fazer que apenas a melhor Pontuação Ranqueada Semanal seja contabilizada, ao invés da pior pontuação em seu lugar.
+3. Participantes não poderão saber da sua posição exata na Colocação de Temporada ou sua pontuação. Ao invés disso, eles irão ser atribuídos as divisões de liga.
+   - Apenas o top 50 da colocação de temporada irá ter sua posição exata visível.
+   - A divisão é atribuída baseado na posição dentro da colocação. Participantes irão ser atribuídos a maior divisão que eles são elegíveis.
+4. A divisão atribuída será revelaso assim que a segunda semana de uma temporada estiver concluída.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:246]:
```suggestion
4. A divisão atribuída será revelada assim que a segunda semana de uma temporada estiver concluída.
```

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 5]
```diff
@@ -0,0 +1,342 @@
...
+
+### Recompensas
+
+*Partes dessas recompensas ainda estão sendo implementadas. Elas Parts serão adicionadas ao longo da temporada. Por causa disso, essa sessão pode mudar a qualquer momento.*
+
+Recompensas são distribuídas para vencedores de salas permanentes semanalmente, criadores dos beatmaps curados e todos os participantes presentes na colocação de temporada.
+
+1. O top 10 de cada sala permanente semanal será recompensado com 1 semana de osu! supporter tags.
+2. Durante a temporada, cada participante é atribuído a uma divisão de liga, que é demonstrado por um distintivo de perfil única que dura pela duração da temporada.
+   - O distintivo irá ser atualizada a cada semana depois da segunda semana de uma temporada.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:256]:
```suggestion
   - O distintivo será atualizado a cada semana depois da segunda semana de uma temporada.
```

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 6]
```diff
@@ -0,0 +1,342 @@
...
+   - Os melhores jogadores de uma temporada talvez poderão manter seu distintivo de perfil permanentemente. Detalhes sobre isso ainda serão decididos.
+3. No final de cada temporada, o melhor criador de beatmap, assim decidido pelos curadores, será recompensado com 1 mês de supporter tag.
+4. No final de cada temporada, uma nova medalha destrancável será adicionada. Jogadores precisarão completar cada beatmap dessa temporada pelo menos uma vez para obter-lá.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:259]:
```suggestion
4. No final de cada temporada, uma nova medalha desbloqueável será adicionada. Jogadores precisarão completar cada beatmap dessa temporada pelo menos uma vez para obtê-la.
```

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 7]
```diff
@@ -0,0 +1,342 @@
...
+Essa tabela mostra apenas uma das quatro versões de distintivos. Cada modo de jogo tem seu próprio distintivo.
+
+### Sistema de curadoria
+
+O sistema de curadoria envolve o processo de curadores selecionando os beatmaps que irão para o Beatmap Spotlights para cada temporada.
+
+1. Beatmaps são selecionados pelos curadores de seus modos de jogo respectivos pela duração de uma temporada.
+   - Curadores precisam concordar com cada dificuldade de beatmap individual em uma discussão aberta.
+   - Líderes de seus modos de jogo respectivos confirmam as decisões e solidificam a seleção após discussões conclusivas.
+   - O processo de seleção em si varia entre os modos de jogo e se ajustam em volta as necessidades de seus membros respectivos.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:281]:
```suggestion
   - O processo de seleção em si varia entre os modos de jogo e se ajustam em volta das necessidades de seus membros respectivos.
```

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 8]
```diff
@@ -0,0 +1,342 @@
...
+2. Beatmaps são selecionados baseado em sua originalidade e excelência. Cada beatmap selecionado deve ser um grande exemplo de qualidade de conteúdo em jogabilidade, design e estética.
+3. Beatmaps curados servem de recomendações para toda a comunidade do osu! em geral.
+4. Para preencher o dever de recomendar excelente beatmaps para toda a comunidade, beatmaps curados devem cubrir um leque específico de dificuldades Hard, Insane e Expert.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:284]:
```suggestion
4. Para preencher o dever de recomendar excelente beatmaps para toda a comunidade, beatmaps curados devem cobrir um leque específico de dificuldades Hard, Insane e Expert.
```

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 9]
```diff
@@ -0,0 +1,342 @@
...
+   - 30% de todos os beatmaps curados devem estar dentro do tier de dificuldade Expert.
+5. Para cada temporada, pelo menos 20 beatmaps devem ser selecionados.
+   - Todos os beatmaps selecionados devem ser Ranqueados.
+   - Se mais beatmaps forem selecionados, a distribuição entre tiers de dificuldade devem ser seguidos.
+   - Curadores podem selecionar múltiplos beatmaps do mesmo set de beatmap.
+6. Beatmaps curados devem ser um mix saudável de conteúdo recente e estabelecido.
+   - Pelo menos 25% das dificuldades de beatmaps curados devem ter sido Ranqueados dentro dos últimos 3 meses antes do começo da temporada.
+7. Curadores individuais não devem recomendar beatmaps que eles tiveram parte em sua criação.
+   - No máximo 25% dos beatmaps selecionados podem ter tal envolvimento de criação dos próprios curadores.
+8. Cada beatmap curado deve ser selecionado antes da temporada começar. Depois do começo de uma temporada, os beatmaps não podem mais serem trocados.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:296]:
```suggestion
8. Cada beatmap curado deve ser selecionado antes da temporada começar. Depois do começo de uma temporada, os beatmaps não podem mais ser trocados.
```

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 10]
```diff
@@ -0,0 +1,342 @@
...
+
+- [Feedback forum thread](https://osu.ppy.sh/community/forums/topics/1101170)
+- `#beatmap-spotlights` no [Servidor de Discord da comunidade do osu!](https://discord.gg/0Vxo9AsejDkGlk3H)
+- `#osu-spotlights` no [Servidor de Discord do osu!dev](https://discord.gg/ppy)
+
+### Se tornando um curador
+
+Qualquer um pode se candidatar a se tornar um curador preenchendo este [formulário](https://spotlights.team/app). Esses formulários abrem entre as temporadas e fecham durante uma temporada em andamento.
+
+Todos os candidatos são analizados pela gerência do projeto e líderes de times e são avaliados baseado em sua proficiência como jogador, mapper, modder e selecionador de mappool de torneio para seu modo de jogo respectivo. Candidatos não precisam ser proficientes em todas essas coisas para serem selecionados, mas ter um leque grande de habilidades certamente ajuda. A lista de curadores é baseada num mix saudável de variantes níveis de habilidade, experiências e proficiências. O número de curadores está sendo mantido propositalmente pequeno e limitado no momento enquanto o projeto se desenvolve.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:311]:
```suggestion
Todos os candidatos são analisados pela gerência do projeto e líderes de times e são avaliados baseado em sua proficiência como jogador, mapper, modder e selecionador de mappool de torneio para seu modo de jogo respectivo. Candidatos não precisam ser proficientes em todas essas coisas para serem selecionados, mas ter um leque grande de habilidades certamente ajuda. A lista de curadores é baseada num mix saudável de variantes níveis de habilidade, experiências e proficiências. O número de curadores está sendo mantido propositalmente pequeno e limitado no momento enquanto o projeto se desenvolve.
```

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 11]
```diff
@@ -0,0 +1,342 @@
...
+   - Cada beatmap curado é selecionado e confirmado antes que a temporada comece.
+   - Após o início de uma temporada, toda o cronograma será anunciado.
+3. Cada temporada é dividida por semana. Cada semana é nomeada com sua própria letra.
+   - Cada letra representa a playlist de um lobby de multiplayer semanal.
+   - Semanas revezam pela temporada inteira e repetem duas vezes. As 2 semanas finais terão as playlists anteriores agrupadas em 2 playlists combinadas.
+4. Quando uma temporada acabar, haverá uma pausa de 3 semanas antes que a nova temporada comece. Durante esse tempo, ajustes ao projeto poderão ser feitos.
+
+### Cronograma (Verão 2020)
+
+Esse cronograma mostra a disponibilidade de salas permanentes da temporada atual.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:200]:
```suggestion
Este cronograma mostra a disponibilidade de salas permanentes da temporada atual.
```
Grammar things

**Review by daviirodrig — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 12]
```diff
@@ -0,0 +1,342 @@
...
+
+Os **Beatmap Spotlights** (também conhecidos como *Spotlights* ou *Ranking Charts*) é um projeto de curadoria recorrente para recomendação e destaque de [beatmaps](/wiki/Beatmaps) pela seu design e gameplay excelentes e únicos. É acompanhado por uma [liga de temporada](#spotlights-seasons) que dá recompensas para todos os jogadores participantes.
+
+**A iteração atual desse projeto está em desenvolvimento e muitos de suas funcionalidades ainda estão sendo implementadas.** Específicos desse sistema, tais como recompensas, rankings sazonais ou funcionalidades de jogabilidade, podem mudar a qualquer momento.
+
+A temporada atual é a temporada de Verão 2020. É considerada a ser o primeiro teste do novo sistema de Beatmap Spotlights.
+
+## Organização
+
+O project do Beatmap Spotlights é mantido por vários membros da comunidade dentre todos os modos de jogo, liderados por um líder de projeto dedicado para essa função.
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:20]:
```suggestion
O projeto Beatmap Spotlights é mantido por vários membros da comunidade dentre todos os modos de jogo, liderados por um líder de projeto dedicado para essa função.
```

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 13]
```diff
@@ -0,0 +1,342 @@
...
+3. Dentro de uma sala permanente, todo mundo compete entre si em uma colocação específica para aquela sala.
+   - Colocações de sala são baseados no sistema de Pontuação Ranqueada. Portanto, apenas os melhores resultados de pontuação em cada beatmap dentro da playlist são contados para criar uma Pontuação Ranqueada semanal para cada jogador.
+   - Cada beatmap de uma playlist conta igualmente para a pontuação na colocação. Dificuldades mais difíceis não irão fornecer mais pontuação do que as mais fáceis.
+   - Pontuação individual de beatmap é baseada no sistema de pontuação do osu!lazer.
+4. O total semanal de Pontuação Ranqueada visível na sala permanente é chamado de "Pontuação Ranqueada Semanal" na colocação.
+5. Assim que uma sala permanente termina, a Pontuação Ranqueada Semanal de cada participante é adicionada a Colocação de Temporada.
+
+### Colocação de temporada
+
+*Essa funcionalidade ainda está sendo implementada. Parte dela serão adicionadas ao longo da temporada. Por causa disso, essa sessão pode mudar a qualquer momento.*
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:236]:
```suggestion
*Essa funcionalidade ainda está sendo implementada. Partes dela serão adicionadas ao longo da temporada. Por causa disso, essa seção pode mudar a qualquer momento.*
```

File: wiki/Beatmap_Spotlights/pt-br.md [hunk 14]
```diff
@@ -0,0 +1,342 @@
...
+2. Participantes poderão possuir apenas uma Pontuação Ranqueada Semanal por playlist.
+   - Jogar a mesma playlist em outra semana irá fazer que apenas a melhor Pontuação Ranqueada Semanal seja contabilizada, ao invés da pior pontuação em seu lugar.
+3. Participantes não poderão saber da sua posição exata na Colocação de Temporada ou sua pontuação. Ao invés disso, eles irão ser atribuídos as divisões de liga.
+   - Apenas o top 50 da colocação de temporada irá ter sua posição exata visível.
+   - A divisão é atribuída baseado na posição dentro da colocação. Participantes irão ser atribuídos a maior divisão que eles são elegíveis.
+4. A divisão atribuída será revelaso assim que a segunda semana de uma temporada estiver concluída.
+
+### Recompensas
+
+*Partes dessas recompensas ainda estão sendo implementadas. Elas Parts serão adicionadas ao longo da temporada. Por causa disso, essa sessão pode mudar a qualquer momento.*
```
Comment (daviirodrig) [inline wiki/Beatmap_Spotlights/pt-br.md:250]:
```suggestion
*Partes dessas recompensas ainda estão sendo implementadas. Elas serão adicionadas ao longo da temporada. Por causa disso, essa seção pode mudar a qualquer momento.*
```

**Review by daviirodrig — CHANGES_REQUESTED** [review]
Missed some strings

File: wiki/Beatmap_Spotlights/pt-br.md (hunk 3 above)
Comment (Flantsune) [reply wiki/Beatmap_Spotlights/pt-br.md:16]: I actually had a lot of trouble trying to figure out something for this one. Thanks!

**Review by daviirodrig — DISMISSED** [review]
Good to go!

**Review by TPGPL — APPROVED** [review]

**Pull request merged**